admins/pageflow/accounts.rb

```diff
@@ -139,5 +139,22 @@ def permitted_attributes_for(resource_name)
         end
       end
     end
+
+    member_action 'confirm_invitations',
+                  title: 'pageflow.admin.users.confirm_invitations', method: :get do
+      account = Pageflow::Account.find(params[:id])
+
+      invitations = account.invitations.by_user(current_user)
+
+      if invitations.empty?
+        redirect_to admin_root_path,
+                    alert: I18n.t('pageflow.admin.users.no_invitations_available')
+      else
+        invitations.turn_into_memberships
+
+        redirect_to admin_root_path,
+                    notice: I18n.t('pageflow.admin.users.invitations_confirmed')
```

Comment:
Please watch out for the correct i18n namespace. The part after `pageflow.admin` should correspond to the enclosing admin.